api/v1alpha1/rekor_types_test.go

```diff
@@ -270,7 +272,9 @@ var _ = Describe("Rekor", func() {
 					Expect(k8sClient.Create(context.Background(), &rekorInstance)).To(Succeed())
 					fetchedRekor := &Rekor{}
 					Expect(k8sClient.Get(context.Background(), getKey(&rekorInstance), fetchedRekor)).To(Succeed())
-					Expect(fetchedRekor.Spec).To(Equal(expectedRekorInstance.Spec))
+					Expect(fetchedRekor.Spec.Pvc.Name).To(Equal(expectedRekorInstance.Spec.Pvc.Name))
+					Expect(fetchedRekor.Spec.Pvc.Size).To(Equal(expectedRekorInstance.Spec.Pvc.Size))
+					Expect(fetchedRekor.Spec.RekorSearchUI.Enabled).To(BeTrue())
```

Comment:
Same here - again my earlier example omitted the deferencing.
```suggestion
					Expect(*fetchedRekor.Spec.RekorSearchUI.Enabled).To(BeTrue())
```